asistente_ladm_col/utils/decorators.py

```diff
@@ -166,6 +166,19 @@ def decorated_function(*args, **kwargs):
 
     return decorated_function
 
+def _grass7_required(func_to_decorate):
+    @wraps(func_to_decorate)
+    def decorated_function(*args, **kwargs):
+        try:
+            import processing
+            grass7 = processing.algs.grass7.Grass7Utils.Grass7Utils
+            print(grass7.installedVersion())
+            return func_to_decorate(*args, **kwargs)
+        except AttributeError:
+            msg = QCoreApplication.translate("AsistenteLADMCOLPlugin", "GRASS not found.")
+            Logger().warning(__name__, msg)
```

Comment:
Do you need the `msg` variable at all?

Reply:
The variable `msg` is not required. It can be omitted.